data/prepare_inputs.py

```diff
@@ -624,7 +920,9 @@ def prep_inputs_for_prediction_data(self):
             with open(f"{out_path}/mean_std_dict.pkl", 'wb') as f:
                 pickle.dump(mean_std_dict, f)
 
-        return 0
+            ak.to_parquet(events, f"{full_path_to_save}/events.parquet")
```

Comment:
We were not saving the output before? I see it was commented out (so now it can be deleted)?